crates/cxx-qt-lib/Cargo.toml

```diff
@@ -22,6 +22,7 @@ http = { version = "1.0", optional = true }
 rgb = { version = "0.8", optional = true }
 time = { version = "0.3.20", optional = true }
 url = { version = "2.3", optional = true }
+serde = { version = "1", features=["default", "derive"], optional = true }
```

Comment:
Why do you need "default" ?

The docs suggest that this should work `serde = { version = "1.0", features = ["derive"] }`

https://serde.rs/derive.html

Reply:
It seems we don't, done :)